server/models/Collective.js

```diff
@@ -2409,6 +2409,28 @@ function defineModel() {
       });
     }
 
+    await sequelize.query(
+      `
+        UPDATE public."Orders" SET status = 'CANCELLED'
+        WHERE id IN (
+          SELECT "Orders".id FROM public."Orders"
+          INNER JOIN public."Subscriptions" ON "Subscriptions".id = "Orders"."SubscriptionId"
+          WHERE
+            "Orders".status = 'ACTIVE' AND
+            "Subscriptions"."isManagedExternally" AND
+            "Subscriptions"."isActive" AND
+            "Orders"."CollectiveId" = ?
+        )
+      `,
+      {
+        type: QueryTypes.UPDATE,
+        replacements: [this.id],
+      },
+    );
```

Comment:
:ice_cream: I would isolate this query somewhere in a `cancelNonTransferableSubscriptions` function, with a comment like: `Cancels all subscription that cannot be transferred when changing hosts (i.e. PayPal)`.

As an alternative if we don't do this, we should at least add a comment to explain why we're canceling only `isManagedExternally`; that might not be clear for someone not familiar with this system.